docs/import/running-the-import.md

```diff
@@ -0,0 +1,76 @@
+# How to run the import pipeline
+
+Configure settings:
+```bash
+# Repository and branch used for the run. Useful for testing on different branches.
+export REPOSITORY=https://github.com/EBIvariation/metadata-ws
+export BRANCH=master
+
+# Path where Maven is installed (or will be installed)
+# See also instructions below
+export MAVEN_INSTALL_PATH=
+export PATH=$MAVEN_INSTALL_PATH/bin:$PATH
+
+# Oracle credentials to download the database drivers
+# See Confluence → Variation home → IT infrastructure → External services credentials → Oracle
+export ORACLE_USERNAME=
```

Comment:
Alright, so this is to login to the Oracle website. At first sight, I thought this was some Oracle database. Could this be renamed to ORACLE_SSO_USERNAME or something similar for clarity?

Reply:
I agree, fixed in https://github.com/EBIvariation/metadata-ws/pull/86/commits/608ed7c7bf51ffab28ecd9637833f8cf5bf94477